src/index.ts

```diff
@@ -47,6 +48,12 @@ export type {
   ScoredPineconeRecord,
 } from './data';
 export type { FetchAPI } from './pinecone-generated-ts-fetch';
+export type {
+  IndexMeta,
+  IndexMetaStatus,
+  IndexMetaStatusStateEnum,
+  IndexMetaDatabase,
+} from './pinecone-generated-ts-fetch';
```

Comment:
The docs build was complaining about these being missing. So I figured I should export them.